challenge_2/ruby/selectivealso/methods.rb

```diff
@@ -0,0 +1,12 @@
+class Array
+  def find_unique
+    count = Hash.new(0)
+
+    self.each do |num|
```

Comment:
Also what do you think about renaming `num` to `element` since you use this later on for Strings.